.github/workflows/update-bridge.yml

```diff
@@ -52,35 +47,33 @@ jobs:
       with:
         python-version: ${{matrix.pythonversion}}
     - run: sudo npm install -g chg
```

Comment:
we don't need this dep installed anymore

Reply:
Fixed.